backend/Makefile

```
@@ -0,0 +1,100 @@
+# Code quality
+lint:
+	flake8 app tests
+
+format:
+	black app tests
+	isort app tests
+
+type-check:
+	mypy app
+
+check: lint type-check test
```

checkmake (0.2.2)

[warning] Target "check" should be declared PHONY.

(phonydeclared)